### ISSUE TITLE
Ghost button contrast

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#424]
From independent accessibility report, SC 1.4.3:

<blockquote lang="nl">
Op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/collapsible.html">components/collapsible.html</a> staat een inklapbare ghost button. De gele tekst heeft een contrast van 1,7:1 met de witte achtergrond.
</blockquote>

The documentation theme (`soft`) uses a text colour for ghost buttons that provides insufficient contrast.
